aliyun/src/main/java/org/apache/iceberg/aliyun/AliyunProperties.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aliyun;
+
+import java.io.Serializable;
+import java.util.Map;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.util.PropertyUtil;
+
+public class AliyunProperties implements Serializable {
+
+  /**
+   * The domain name used to access OSS. OSS uses HTTP Restful APIs to provide services. Different regions are accessed
+   * by using different endpoints. For the same region, access over the internal network or over the Internet also uses
+   * different endpoints. For more information, see:
+   * https://www.alibabacloud.com/help/doc-detail/31837.htm?spm=a2c63.p38356.879954.7.5dcb7d9186qY48#concept-zt4-cvy-5db
+   */
+  public static final String OSS_ENDPOINT = "oss.endpoint";
+
+  /**
+   * OSS uses an AccessKey pair, which includes an AccessKey ID and an AccessKey secret to implement symmetric
+   * encryption and verify the identity of a requester. The AccessKey ID is used to identify a user.
+   * <p>
+   * For more information about how to obtain an AccessKey pair, see:
+   * https://www.alibabacloud.com/help/doc-detail/53045.htm?spm=a2c63.p38356.879954.8.5dcb7d9186qY48#task968
```

Comment:
I think the `spm=a2c63.p38356.879954.8.5dcb7d9186qY48#task968` be removed to make the URL cleaner, similar comment for all the URLs below.

Reply:
Yeah,   those parameters could be removed.  Thanks for the reminding !  @jackye1995